src/pages/references/changelog.md

```diff
@@ -21,6 +21,24 @@ contributors:
 
 # Changelog
 
+## 2024-09-30
+
+- Added many new **Text APIs** for improved text management.
+  - New [`TextNode.fullContent`](./document-sandbox/document-apis/classes/TextNode.md#fullcontent) accessor: returns the [`TextContentModel`](./document-sandbox/document-apis/classes/TextContentModel.md) containing the complete text string and its styles. `fullContent.text` supersedes `TextNode.text`, now deprecated.
+  - New [`TextNode.layout`](./document-sandbox/document-apis/classes/TextNode.md#layout) accessor: gets and sets the [`TextType`](./document-sandbox/document-apis/enumerations/TextType.md) of the text node frame.
+  - New [`TextNode.nextTextNode`](./document-sandbox/document-apis/classes/TextNode.md#nexttextnode) accessor: gets the newxt node that overflowing text will spill into.
```

Comment:
Typo 'newxt'. 
Can we move this to the second place so that all threaded text related points are next to each other?

Reply:
Done!